### PR TITLE
Change the API route for admin repocheck

### PR DESCRIPTION
**Pull request by Itxaka**
Due to the movement of the admin repocheck to the upgrade namespace,
we need to change the route we use to call the api

Depends on https://github.com/crowbar/crowbar-core/pull/739
- Move the call to the admin repo check to the proper namespace
- Move the fake API route to the proper namespace

### CODE REVIEW

**Comment by Itxaka** [comment]
[(3) S49P19 #32 Admin Server Repo Checks: Service Integration](https://trello.com/c/oGniN5Lf/149-3-s49p19-32-admin-server-repo-checks-service-integration)

File: assets/app/data/crowbar/services/crowbar.factory.js [hunk 1]
```diff
@@ -41,7 +41,7 @@
```
Comment (santiph) [inline assets/app/data/crowbar/services/crowbar.factory.js]: (for some reason I cannot add this comment on line 12)
This method (`getRepositoriesChecks`) needs to be moved from crowbar factory to upgrade factory, as they're intended to be a reflection of the API entities.

**Review by santiph — CHANGES_REQUESTED** [review]
Crowbar Factory method and its test needs to be moved to the Upgrade Factory.

File: assets/app/data/crowbar/services/crowbar.factory.js (hunk 1 above)
Comment (Itxaka) [reply assets/app/data/crowbar/services/crowbar.factory.js]: out of scope for this patch or wrong PR?

**Comment by Itxaka** [comment]
Ready to review as dependant patch has been merged!

File: assets/app/data/crowbar/services/crowbar.factory.js (hunk 1 above)
Comment (santiph) [reply assets/app/data/crowbar/services/crowbar.factory.js]: I don't think so.
Repository checks for the admin server now is part of the upgrade API entity, not crowbar.
Therefore, in order to keep consistency, I think we should reflect that by moving the `getRepositoriesChecks` method from crowbar.factory.js into upgrade.factory.js

Comment (Itxaka) [reply assets/app/data/crowbar/services/crowbar.factory.js]: Ahhh I see now! Indeed you are rigth, this should be moved into the appropiate place now that has changed places in the API

**Review by santiph — APPROVED** [review]

**Comment by santiph** [comment]
Looks good to me now. :) Thanks!

**Pull request merged**